### ISSUE TITLE
"app_process: not found" since updating to 0.31

### ISSUE THREAD

**Issue by benjaminoakes** [termux/termux-app#65]
I'm getting this message when using any Termux API calls (such as  `termux-notification`, `termux-toast`, and possibly others):

    /system/bin/am[8]: app_process: not found

That makes a number of my scripts break -- any that use the `termux-*` commands.

I'm also having issues updating using `apt`.  It freezes like this:

    Get:1 http://termux.net stable InRelease [1470 B]
    Get:2 http://termux.net stable/main all Packages [1932 B]
    Get:3 http://termux.net stable/main arm Packages [37.9 kB]
    93% [Working]

(It never gets goes past 93%.)  However, that was the case since before v0.31. Perhaps I have an old version of the Termux API utilities?  If so, I can't update them because of the update issue.  Do I need to completely uninstall the app set up everything again?

**Comment by Neo-Oli** [comment]
This has been fixed a few minutes ago.
 https://github.com/termux/termux-api/issues/12

The 93% Problem was something that happened a few weeks ago. https://plus.google.com/+RashidDavid/posts/3R4P1qsWRoW
You need to manually update apt. 

For arm cpu(likely):
```
wget http://termux.net/dists/stable/main/binary-arm/apt_1.2.3_arm.deb
dpkg -i apt_1.2.6_arm.deb
```

for intel
```
wget http://termux.net/dists/stable/main/binary-i686/apt_1.2.6_i686.deb
dpkg -i apt_1.2.3_i686.deb﻿
```
Then you can run `apt update && apt upgrade` to update the termux-api.

**Comment by fornwall** [comment]
Hi, the termux-api package has been fixed in the newly released version 0.13 of the package.

However, your `apt` hanging means that you cannot install this update :). Can you do the following steps:

1. Run `rm $PREFIX/var/lib/apt/lists/termux.net*`
2. Run `apt-get clean`.
3. Run `apt update` and see if the problem remains.
4. (If the problems remain): Paste the output of running `apt --version` and `apt update -o Debug::Acquire::http=true` here!

**Comment by benjaminoakes** [comment]
Manually updating `apt` and then running `apt update && apt upgrade` seems to have taken care of it.  Thanks!
